integration/openshift/README.md

```diff
@@ -0,0 +1,27 @@
+# Openshift integration with Clear Containers
```

Comment:
Replace Openshift with OpenShift\*
The * is to satisfy the trademark in the first mention.